client/src/pages/organizations/Show.js

```diff
@@ -356,44 +356,16 @@ const OrganizationShow = ({ pageDispatchers }) => {
                 action={action}
               />
               <Fieldset id="info">
-                <Field
-                  name="status"
-                  component={FieldHelper.ReadonlyField}
-                  humanValue={Organization.humanNameOfStatus}
-                />
-
-                <Field
-                  name="type"
-                  component={FieldHelper.ReadonlyField}
-                  humanValue={Organization.humanNameOfType}
-                />
-
-                {renderLeadingPositions(
-                  organization.positions,
-                  PositionRole.LEADER.toString(),
-                  pluralize(
-                    utils.titleCase(PositionRole.LEADER.humanNameOfRole())
-                  )
-                )}
-
-                {renderLeadingPositions(
-                  organization.positions,
-                  PositionRole.DEPUTY.toString(),
-                  pluralize(
-                    utils.titleCase(PositionRole.DEPUTY.humanNameOfRole())
-                  )
-                )}
-
                 <LongNameWithLabel
                   dictProps={orgSettings.longName}
                   name="longName"
                   component={FieldHelper.ReadonlyField}
                 />
 
-                <IdentificationCodeFieldWithLabel
-                  dictProps={orgSettings.identificationCode}
-                  name="identificationCode"
+                <Field
```

Comment:
It makes sense to keep the same order of fields on the organization Form.

Reply:
There are sections that are shown or not the same in some form and show page or preview structures, or even if they are the same, I have adjusted them to be more comfortable and more useful to the eye.